### PR TITLE
WINGS ARE NOW ORGANS HOLY SHIT

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
WINGS ARE NOW ORGANS! YOU CAN SURGICALLY REMOVE AND IMPLANT WINGS. 
image
FLIGHT POTION NOW WORKS ON ALL RACES
image
image
also, apid changes due to wing changes. Apids no longer wear boots in their neck slots and call them wings. Apids have had their downside lowered, and their dash removed- a roundstart species with a combat ability shouldn't have been merged to begin with (it appears qwerty merged it, and he's been seen to merge prs of questionable balance before... i mean, these fucks were literally wearing boots on their necks!). Apids can now open and close their wings as the original coder seemed to have intended, by using *flap or *wing

## Why It's Good For The Game

holy shit wings are shitcode as fuck

## Changelog
:cl: Zeskorion, AzlanonPC, thatguythere03
add: Wings are now organs!  they can be surgically removed and attached, allowing doctors to create even more hideous abominations!
add: cybernetic wings, unlocked in shuttle technodes. A normal version, equivalent to normal wings, and an alien version that works without atmosphere exist
tweak: flight potion now works on all races, and no longer removes the backpack slot. if the imbiber has wings already, their wings will be upgraded instead, and keep their original look
balance: apid wings now work like moth wings, instead of having a special dash that has combat utility, and their downside has been reduced to 1.5X burn damage, and keeping their flyswatter and pest spray penalties
fix: apids can open and close their wings again. the *wing emote can be used to toggle them, and the *flap and *aflap emotes make them flap
code: makes flight code less shtity and snowflaky
imageadd: moth wing sprites by azlanonpc
imageadd: shitty apid wing codersprites by zesko
imageadd: cybernetic wings by thatguythere03
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
do not merge label is due to pending sprites for the wing items- they all look like grey tails right now

**Comment by YakiAttaki** [comment]
i have a question:
does this mean a moth being cloned with burnt wings will regain their wings?

**Comment by ghost** [comment]
> balance: apid wings now work like moth wings, instead of having a special dash that has combat utility, and their downside has been reduced to 1.5X burn damage, and keeping their flyswatter and pest spray penalties

😔

**Comment by zeskorion** [comment]
> i have a question:
> does this mean a moth being cloned with burnt wings will regain their wings?

Maybe

**Comment by Compilatron144** [comment]
Holy shit plasmemes with fucking dragon wings

Shame their wings don't combust ):

**Comment by YakiAttaki** [comment]
it does not mean moths get their og wings back, i believe this is because when your wings are burnt as a moth the sprite state changes

**Comment by zeskorion** [comment]
ready for merge

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 1]
```diff
@@ -210,7 +214,9 @@
 		/obj/item/organ/vocal_cords/adamantine = 1,
 		/obj/effect/gibspawner/xeno = 1,
 		/obj/effect/mob_spawn/human/corpse/assistant = 1,
-		/obj/effect/mob_spawn/teratomamonkey = 5)
+		/obj/effect/mob_spawn/teratomamonkey = 5,
+		/obj/item/organ/wings/moth/robust = 1,
+		/obj/item/organ/wings/dragon)
```
Comment (PowerfulBacon) [inline code/game/objects/effects/spawners/lootdrop.dm]:
```suggestion
		/obj/item/organ/wings/dragon = 1)
```
This should probably have a number like all the other ones

File: code/game/objects/items/devices/scanners.dm [hunk 2]
```diff
@@ -359,8 +359,6 @@ GENE SCANNER
 			mutant = TRUE
 		else if (S.mutantstomach != initial(S.mutantstomach))
 			mutant = TRUE
-		else if (S.flying_species != initial(S.flying_species))
```
Comment (PowerfulBacon) [inline code/game/objects/items/devices/scanners.dm:362]: Should you not replace this with mutant_wings check to see if they are a mutant?

**Review by PowerfulBacon — DISMISSED** [review]
This is good, but the changes to apids are really bad, since you have made them into basically just reskinned moths. Apids should keep their ability to do the apid dash.

**Comment by PowerfulBacon** [comment]
@ike709 Assigned cause needs testmerging

**Comment by Compilatron144** [comment]
F

File: code/game/objects/items/devices/scanners.dm (hunk 2 above)
Comment (zeskorion) [reply code/game/objects/items/devices/scanners.dm:362]: that is unecessary. This checks if the user has their species organ var changed, which doesnt ever happen with wings. it does not check their actual organs.


besides, scanner gate is never, ever used, ever.

**Comment by yorii** [comment]
I'm so EXCITED 😄

**Comment by ike709** [comment]
No bugs reported after 2 days of testmerge. That doesn't really mean anything since bee players suck at reporting bugs, but whatever.

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm [hunk 3]
```diff
@@ -23,8 +23,6 @@
 	liked_food = GROSS | MEAT
 	inert_mutation = FIREBREATH
 	deathsound = 'sound/voice/lizard/deathsound.ogg'
-	wings_icon = "Dragon"
-	species_language_holder = /datum/language_holder/lizard
```
Comment (yorii) [inline code/modules/mob/living/carbon/human/species_types/lizardpeople.dm:27]: You clumsy!

**Pull request merged**